packages/hs-react-ui/package.json

```diff
@@ -118,7 +122,47 @@
     "clean": "rm -rf ./dist/*",
     "webpack": "webpack --config webpack.config.js",
     "dts": "tsc --declaration --outDir dist/types/ --emitDeclarationOnly --declarationMap --allowJs false --checkJs false",
-    "test": "jest --collectCoverage=true"
+    "pre-push": "yarn run lint-ci && yarn run test && yarn run clean && yarn run webpack && yarn run dts",
+    "test": "rm -rf ./coverage/* && jest --collectCoverage=true",
```

Comment:
Does jest not do this already? (removing /coverage directory) Bummer.

Reply:
I thought it was supposed to, but it didn't look like it was. It DOES overwrite the old ones, but for consistency sake across the scripts for generating the storybook docs and the coverage info